packages/@okta/vuepress-site/guides/custom-hosted-signin/customization-examples/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Customization Examples
+---
+Use the following examples to help you customize your sign-in page with your own CSS, scripts, and per-application branding.
+
+## Add Your Own Stylesheet
```

Comment:
FYI: I removed the HTML banner example because it was much less useful than these. We can always add examples later, for now I've decided to just stick with the most common things.